test/failing/single_line.mli

```diff
@@ -0,0 +1,6 @@
+[@@@ocamlformat "module-item-spacing=compact"]
+
+val xx_xxxxxxxx : t -> bool
+val xx_xxxxxxxx : t -> bool
+val xxxxxxxx : t -> [> `Xxxxxxx | `Xxxxxxxxxxx | `Xxxxxxxxxx | `Xxxxxxxxxxxxx]
```

Comment:
This is exceeding the margin, this is why `is_single_line` was `false`.
I think we should comment in the code that the `+ 1` is a workaround a bug.